Xabe.FFmpeg/Enums/FileExtensions.cs

```diff
@@ -1,13 +1,9 @@
-﻿using System;
-using System.ComponentModel;
-using System.Reflection;
-
```

Comment:
I don't think mp3 or Png is video file

Reply:
I guess it doesn't matter - it is file extensions class not video file extensions.